ci/integration.js

```diff
@@ -0,0 +1,236 @@
+const child_process = require("child_process");
+const { randomBytes } = require("crypto");
+const path = require("path");
+const fs = require("fs");
+
+global.XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+
+const { ajax } = require("rxjs/ajax");
+
+const rxJupyter = require("rx-jupyter");
+
+const _ = require("lodash");
+
+console.log("running bookstore integration tests");
+
+async function genToken(byteLength = 32) {
```

Comment:
I like how I went a little extra by using `randomBytes` then just encoded the minio secrets in the config out of what I expected to be a necessity.

Reply:
Nice!